src/core/backend_context.cc

```diff
@@ -210,14 +211,25 @@ BackendContext::CopyBuffer(
     memcpy(dst, src, byte_size);
   } else {
 #ifdef TRTIS_ENABLE_GPU
+    // [TODO] Add check for Peer Access between two GPUs and use
```

Comment:
Not clear if we need to check for peer access here, the copy is necessary as we want data to be in contiguous chunk for input and output to be moved to allocated buffer.

Reply:
Removing Peer Access comment for now. This is true, a memcpy would be necessary